app/dashboard/trainee/profile/index.tsx

```diff
@@ -66,12 +67,57 @@ export default function Profile() {
       setProfile(data.getProfile);
     }
   }, [data]);
+  
+  useEffect(() => {
+    const fetchOrgToken = async () => {
+      try {
+        const orgToken = await AsyncStorage.getItem('orgToken');
+        if (orgToken) {
+          setOrgToken(orgToken);
+        } else {
+          toast.show('Token Not found.', { type: 'danger', placement: 'top', duration: 3000 });
+        }
+      } catch (error) {
+        toast.show('Failed to retrieve token.', { type: 'danger', placement: 'top', duration: 3000 });
+      } 
+    };
+    fetchOrgToken();
+  }, []);
+
+ const { data: traineedata, error: err } = useQuery(GET_TRAINEE_PROFILE, {
+  context: {
+    headers: {
+      Authorization: `Bearer ${userToken}`,
+    },
+  },
+  skip: !userToken,
+  variables: {
+    orgToken: orgToken,
+  },
+});
+
+useEffect(() => {
+  if (err) {
+    toast.show(`Error fetching profile.${err}` , { type: 'danger', placement: 'top', duration: 3000 });
+  }
+}, [err]);
+
+useEffect(() => {
+  if (traineedata) {
+    setTraineeProfile(traineedata.getProfile);
+  }
+}, [traineedata]);
 
   return (
     <View>
       <View className="relative h-48">
-        <CoverImage cover={profile?.cover} name={profile?.name} />
-        <TouchableOpacity className="absolute bottom-1 right-4 rounded-full shadow-md">
+        <Image
+          source={
+            profile.cover ? { uri: profile.cover } : require('@/assets/images/background.png')
+          }
+          className="w-full h-full object-cover"
+        />
+        <TouchableOpacity onPress={() => router.push('/dashboard/trainee/profile/edit')} className="absolute bottom-1 right-4 rounded-full shadow-md">
           <SvgXml xml={editBG} />
```

Comment:
Revert these changes, both the cover image and edit profile routes were updated

Reply:
Resolved